fracridge/fracridge.py

```diff
@@ -319,7 +323,7 @@ def fit(self, X, y, sample_weight=None):
     def predict(self, X):
         X = check_array(X, accept_sparse=True)
         check_is_fitted(self, 'is_fitted_')
-        if len(self.coef_.shape) == 0 or X.shape[-1] == 1:
+        if len(self.coef_.shape) == 0:
```

Comment:
Wouldn't the code fail to address the single regressor use-case with this change?

Reply:
I think that the single regressor case fails because of the squeeze this pull request removes. The original squeeze call squeezes the (1,) coef_ array into a scalar. This doesn't happen anymore.

Reply:
Ah yes. And I see that the tests are now also passing!